scripts/tests/test_phase44d_carry_aware_experimental_family_matrix_aggregate.py

```
@@ -0,0 +1,352 @@
+                            verify_ms = "0.000"
+                        writer.writerow(
+                            self.sample_row(
+                                variant=variant,
+                                benchmark_version=spec.benchmark_version,
+                                semantic_scope=spec.semantic_scope,
+                                verify_ms=verify_ms,
+                            )
+                        )
+                inputs.append(path)
+
+            completed = subprocess.run(
```

Ruff (0.15.11)

[error] `subprocess` call: check for execution of untrusted input

(S603)